### DISCUSSION TITLE
n1新配置的添加阿里云盘提示错误

### DISCUSSION THREAD

**Discussion by dreamyer** [AlistGo/alist#3950] [Q&A]
Failed init storage: Post "https://open.aliyundrive.com/adrive/v1.0/user/getDriveInfo": dial tcp: lookup open.aliyundrive.com on [**1]:53: server misbehaving

第一次配置不知道是什么问题

**Comment by nvidiaction** [comment]
更新最新的alist，我是这样解决的
